go.mod

```diff
@@ -51,6 +51,7 @@ require (
 	github.com/riywo/loginshell v0.0.0-20190610082906-2ed199a032f6
 	github.com/sanathkr/yaml v1.0.0 // indirect
 	github.com/sirupsen/logrus v1.4.2 // indirect
+	github.com/spf13/afero v1.2.2
```

Comment:
You will need to bump the dependency image version in the image and push it to the registry before merging

Reply:
PS: we need to automate this but I haven't had the time yet

Reply:
Can we make the image tag a hash of dependencies? There is a way to make a git hash but I don't recall what it takes to do, otherwise just a sha sum of the content would seem sufficient?

Reply:
wouldn't it then be more tricky to do this process? because we need to replace that hash in the Makefile. Unless it is done automatically... but then it's not so easy to know if a given image version is what you need when added a new dependency :/

Reply:
Yeah, I was thinking of automatic hashing. If we use git hashes, you can use 'git show <hash>' to tell what went in. I mean tree hashes, not actual commits.

Reply:
#1145